urbanopt/geojson-modelica-translator
# Cli

#### Any background context you want to provide?
- This is branched off the `write-sys-params` branch, so it includes updates from #250. That PR should be merged first.
- The uo_des CLI currently assumes the user has already run simulations with the uo sdk CLI.
    - Perhaps at some point in the future one cli will become the primary and call the other cli to accomplish tasks.
#### What does this PR accomplish?
Creates a user interface for the GMT. At this time only running the CSVToSysParam class
#### How should this be manually tested?
- `pip install .` (to install the new cli from setup.py)
- `uo_des --help`

Expect to see help text with list of available cli commands, currently only `sys-param`
For further details, help text for the command can be seen with `uo_des sys-param --help`
#### What are the relevant tickets?
#254 
#### Screenshots (if appropriate)
